libcxx/test/std/atomics/atomics.ref/load.pass.cpp

```diff
@@ -0,0 +1,55 @@
+//
```

Comment:
I think majority of the tests would still pass even if the implementation does not have any atomic operations. i would suggest to have a look at these tests
https://github.com/llvm/llvm-project/blob/main/libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/test_helper.h

(most of the tests in atomic float uses the test template above)